sr_description/hand/xacro/backward_compat.urdf.xacro

```diff
@@ -0,0 +1,46 @@
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+  <!-- for backward compatibility define new params from old ones -->
+  <xacro:property name="old_tip_sensor" value="${(1 if bio==True else 0) + (1 if bt_sp==True else 0) + (1 if ubi==True else 0) + (1 if eli==True else 0)}"/> 
+  <!-- if old parameters are used, valid only if sum is 1 -->
+  <xacro:if value="${old_tip_sensor==1}">
+    <xacro:if value="${bio}">
+      <xacro:property name="palm_sensor" value="std"/>
+      <xacro:property name="prox_sensor" value="${[std, std, std, std ,std]}"/>
```

Comment:
This lists are ill-formed. These should be lists of strings.
```suggestion
      <xacro:property name="prox_sensor" value="${['std', 'std', 'std', 'std', 'std']}"/>
```

Reply:
@rhaschke thanks  for spotting a bug in #109 , and sorry @beatrizleon  if it affected you. 
I would like to clarify the claims made above. The backward compatibility was tested first, including a check of most significant if not all files for diff with previous version before the refactor, and including the generation of all backward compatibility files against expansion. 
So the translation was process was really triggered.

Then a late change came up on 2 place in the main code, that would have required the backward compatibility to be updated and retested which was only partially done (called the test but not a diff with old files before refactor). 
* renaming attributes from singular to plural made the backward compatibility variables unused, and this not processed by xacro due to lazy evaluation.
* change in the content of the attributes, from single names to vectors, that requires a vector of strings. The correct vector of strings is used in the main code, but was forgotten to be fixed in the backward compat and did not show up as ill-defined due to the previous point.

So my apology again to not have re-verified the backward compatibility was working when doing the PR.
I know fixing this PR makes not much more sense now that there is a new PR #120 but I like to understand what happens, and clarify things.